roaster-tm4j/roaster-tm4j-dto/src/main/resources/xsd/hu/icellmobilsoft/roaster/tm4j/dto/domain/test-execution.xsd

```diff
@@ -0,0 +1,19 @@
+<xs:schema xmlns="http://tm4j.dto.roaster.icellmobilsoft.hu/test-execution"
+           xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           targetNamespace="http://tm4j.dto.roaster.icellmobilsoft.hu/test-execution"
```

Comment:
http://tm4j.dto.roaster.icellmobilsoft.hu/test-execution nem tukrozi a konyvtarat, fel van cserelve a dto es tm4j, a a konyvtarban van `domain` is